styles.css

```diff
@@ -14,4 +14,18 @@ p {
   width: 400px;
   margin: 50px auto;
   font: 30px Monaco,"Courier New","DejaVu Sans Mono","Bitstream Vera Sans Mono",monospace;
```

Comment:
{   "ns": "yt",   "el": "embedded",   "cpn": "DiUW10ja5iFU98cv",   "ver": 2,   "cmt": "31.104",   "fmt": "244",   "fs": "0",   "rt": "74.777",   "euri": "https://blog.google/",   "lact": 12,   "cl": "667424570",   "mos": 0,   "state": "8",   "volume": 100,   "cbrand": "generic",   "cbr": "Chrome Mobile",   "cbrver": "127.0.0.0",   "c": "WEB_EMBEDDED_PLAYER",   "cver": "1.20240825.00.00",   "cplayer": "UNIPLAYER",   "cmodel": "android 10.0",   "cos": "Android",   "cosver": "10",   "cplatform": "MOBILE",   "autoplay": "1",   "epm": 1,   "splay": "1",   "hl": "en_US",   "cr": "US",   "len": "100.301",   "fexp": "v1,24004644,434717,127326,26443548,7111,24166,12177,9954,1192,33462,2,9358,8410,29151,2197,9996,1103,6953,11044,501,1970,7546,12724,5077,12945,11076,2,2126,1336,4204,1,1823,3186,1240,1672,4794,4,9,2,1020,1739,328,13,3805,23,4,403,649,932,3476,4115,31,4255,544,1303,273,8",   "afmt": "251",   "size": "320:180:3.375",   "inview": "0",   "muted": "0",   "conn": "7",   "docid": "s4InWsd-J6g",   "ei": "FVfOZpSGBbi_lu8PqdtV",   "plid": "AAYgsgBF5OJh9q96",   "referrer": "https://www.youtube-nocookie.com/embed/s4InWsd-J6g?enablejsapi=1&origin=https%3A%2F%2Fblog.google&widgetid=1",   "of": "wbAUJoLFDaeeOSJrHXH1sg",   "vm": "CAEQABgEOjJBSHFpSlRJdldwQl9lNzJsQ2ZFUzNiM1BkUTlJel9GVU50bElfclBlck5CZFhDWHExUWJsQU9BckJGdjJ4Y3oxaVhyaWhrbVJqdFVGWlkxQVM5Z1NLNzdfVXVjWmxPdllWWDRKWWV4M1V2czNOb1p6Z1BZRVVsTGFWd2hFZUg1Vk1LdzRfMlc5dlUxejRBY2tCVE40RWI0MVhKMUlQRUFV",   "vct": "31.104",   "vd": "100.301",   "vpl": "0.000-31.104",   "vbu": "0.000-79.779",   "vbs": "0.000-100.301",   "vpa": "0",   "vsk": "0",   "ven": "0",   "vpr": "1",   "vrs": "4",   "vns": "2",   "vec": "null",   "vemsg": "",   "vvol": "1",   "vdom": "1",   "vsrc": "1",   "vw": "320"https://cloud.withgoogle.com/next?utm_source=copylink&utm_medium=unpaidsoc&utm_campaign=FY24-Q2-global-ENDM33-physicalevent-er-next-2024-mc&utm_content=next-homepage-social-share&utm_term=-   "vh": "180",